packages/hawtio/src/util/strings.ts

```diff
@@ -93,3 +93,8 @@ export function stringSorter(a: string, b: string): number {
   }
   return 0
 }
+
+export function parseBoolean(value: string) {
```

Comment:
unless you declare return type, shouldn't it become any? then it could do more than expected, which should be avoided.

Reply:
I don't know if I understood your message correctly. If you meant that by not putting `: boolean` there it might return anything (like an object) yes, you are right, and this should have return type of `boolean` on my opinion.

Setting types also allows TS to do type inference correctly. It might require a `as boolean` in the JSON.parse to conform to type checkin tho

Reply:
Refactored and tests added

Reply:
What I expected is defining return type explicitly like this:
```
export function parseBoolean(value: string): boolean {
```
The current impl is clear that it indeed returns boolean, but still the func signature relies on type inference. I expect that every exported function, const, type, class, etc. has clear type definitions unless it doesn't return anything (`void`). It's ok to omit them for private ones for simplicitly, because the scope is narrow so less chances of confusion.

Reply:
Occam's razor.
In this case, I updated the fn signature
Then thought I would no longer need it. 
Then realised I would and so `git checkout -f string.ts` to bring it back.
Then changed the body but not the signature.

Anyway, good catch and signature no updated.